generatorcore/generator.py

```diff
@@ -86,7 +86,6 @@ class Result:
     h30: heat2030.H30
     l30: lulucf2030.L30
     a30: agri2030.A30
-    h30: heat2030.H30
```

Comment:
Duplicated declaration: https://github.com/GermanZero-de/localzero-generator-core/pull/202/files#diff-944ddb27cc5630d712e60edf1933d707583fd2a017f22d23a3f1696939e55bc2R86

Reply:
One more note -- while I'm redoing your changes I had to learn that pyright in strict mode does NOT seem to report this error. That's disappointing. I'll add investigating more into pyright's config on my stack.